cli/azd/pkg/input/console.go

```diff
@@ -1056,9 +1056,9 @@ func NewConsole(
 // IsTerminal returns true if the given file descriptors are attached to a terminal,
 // taking into account of environment variables that force TTY behavior.
 func IsTerminal(stdoutFd uintptr, stdinFd uintptr) bool {
-	// User override to force non-TTY behavior
-	if ok, _ := strconv.ParseBool(os.Getenv("AZD_DEBUG_FORCE_NO_TTY")); ok {
-		return false
+	// User override to force TTY behavior
+	if forceTty, err := strconv.ParseBool(os.Getenv("AZD_FORCE_TTY")); err == nil {
```

Comment:
Perhaps it'd be nice to also make the TTY column width specifiable, here's what our friends at `gh` did for inspiration: https://github.com/cli/cli/pull/4146/files

Reply:
We don't have any output that is width-dependent currently (aside from the spinner). The spinner itself writes new-line messages when TTY-behavior is off. So there isn't a need right now.

In the future, perhaps we could still have this as an integer, but `1` or `0` is treated as boolean but other numbers are treated as width. Or we could introduce units: `1000w` and `200%`.